binaries/apiserver/main.go

```diff
@@ -29,25 +32,59 @@ func main() {
 		log.Fatal(err)
 	}
 
+	type StoreAndHandler struct {
```

Comment:
I don't know why you need this type; it seems really clumsy. What are you trying to do with it?